README-code-reviews.md

```diff
@@ -0,0 +1,103 @@
+Code Review Policies
+====================
+
+Project Stewardship
+-------------------
+
+The OpenJFX Project is guided by the Project Leads and Reviewers.
+
+__Project Co-Lead__: Kevin Rushforth (kcr) <br>
+__Project Co-Lead__: Johan Vos (jvos)
+
+### Reviewers
+
+The [List of Reviewers](https://openjdk.java.net/census#openjfx) is on the OpenJDK Census.
+
+Overview
+--------
+
+All code must be reviewed before being pushed to the repository. The short version of the OpenJFX code review policy is:
+
+* We define a formal "Reviewer" role, similar to the JDK project, and outline the responsibilities of Reviewers
+* Reviewers, PR authors, and sponsoring Committers verify the following before integration:
+    * All feedback from reviewers is addressed
+    * All Reviewers who have asked for the chance to review have done so (or indicated that they no longer need to)
+    * Sufficient time (at least 1 business day) has passed to allow others to comment
+* The code review policies recognize the following different types of changes, with different minimum thresholds for review:
+    * Simple, low-impact fixes: 1 reviewer
+    * Higher-impact fixes: 2 reviewers
+    * Features / API changes: CSR for approving the change, including approval by a "lead"; implementation then needs 2 reviewers for the code (as with other "higher-impact" fixes above)
+
+Details
+-------
+
+Code reviews are important to maintain high-quality contributions, but we recognize that not every type of change needs the same level of review. Without lowering our standards of quality, we want to make it easier to get low-impact changes (simple bug fixes) accepted.
+
+In support of this, the following review policies are in effect. Many of these will involve judgment calls, especially when it comes to deciding whether a fix is low impact vs. high-impact, and that's OK. It doesn't have to be perfect.
+
+### 1. The Reviewer role for the OpenJFX Project
+
+We define a formal "Reviewer" role, similar to the JDK project. A [Reviewer](https://openjdk.java.net/census#openjfx) is responsible for reviewing code changes and helping to determine whether a change is suitable for including into OpenJFX. We expect Reviewers to feel responsible not just for their piece, but for the quality of the JavaFX library as a whole. In other words, the role of Reviewer is one of stewardship. See the following section for what constitutes a good review.
```

Comment:
(https://openjdk.java.net/census#openjfx)

.org please

BTW these very long source lines make it awkward to precisely identify the text I'm commenting on.

Reply:
I'll fix it.

Reply:
fixed